### ISSUE TITLE
Update floxdocs to be WCAG AA compliant

### ISSUE THREAD

**Issue by zmitchell** [flox/floxdocs#5]
### Acceptance Criteria

* floxdocs passes Axe plugin validation for WCAG AA
   * change light pink color on light mode pages to `#E5006E`
   * change text color to `#1B1B1B` from `#666666`
   * Address aria label related issues

### Original issue

The current typeface color is a mid-gray on a white background, which gives the text low contrast against the background. Low contrast is hard on people with poor eyesight. We should make the body text closer to black to make it easier to read.
